php/antidot/composer.json

```diff
@@ -12,11 +12,11 @@
     "ext-intl": "*",
     "ext-json": "*",
     "antidot-fw/framework": "~0.2.1",
+    "antidot-fw/react-framework": "~1.1",
```

Comment:
Added at least the latest antidot react integration version for better performance.

Reply:
now that reactphp is > 1, might you use `@stable` ?